templates/settings.html

```diff
@@ -16,7 +16,7 @@ <h2>User preferences</h2>
 
 {% block content %}
 <section>
-
+  
```

Comment:
It looks like you added some whitespace here that is not needed.

Reply:
I added it back after accidentally removing it.  But i'll remove it again, both at the beginning of the section and at the end.